benchmarks/activerecord/Gemfile.lock

```diff
@@ -32,23 +33,25 @@ GEM
     minitest (5.25.5)
     mutex_m (0.3.0)
     securerandom (0.4.1)
-    sqlite3 (1.7.3)
+    sqlite3 (2.7.3)
       mini_portile2 (~> 2.8.0)
+    sqlite3 (2.7.3-x86_64-linux-gnu)
     timeout (0.4.3)
     tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
+    uri (1.0.3)
 
 PLATFORMS
   ruby
   x86_64-linux
 
 DEPENDENCIES
-  activerecord (~> 7.2)
+  activerecord (~> 8.0)
   activerecord-jdbcsqlite3-adapter (~> 70)
   base64
   bigdecimal
   mutex_m
-  sqlite3 (~> 1.4)
+  sqlite3
 
 BUNDLED WITH
-   2.8.0.dev
```

Comment:
Please avoid downgrading the Bundler version https://github.com/Shopify/yjit-bench/pull/371. The Bundler bug has not been fixed yet.

Reply:
Fixed in https://github.com/Shopify/yjit-bench/pull/378